<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="33857" started-at="2020-06-19T06:35:12Z" finished-at="2020-06-19T06:35:45Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="33857" started-at="2020-06-19T06:35:12Z" finished-at="2020-06-19T06:35:45Z">
      <class name="com.actitime.testscripts.CustomerTest">
        <test-method status="PASS" signature="openBrowser()[pri:0, instance:com.actitime.testscripts.CustomerTest@4501b7af]" name="openBrowser" is-config="true" duration-ms="4636" started-at="2020-06-19T06:35:12Z" finished-at="2020-06-19T06:35:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="PASS" signature="loginApp()[pri:0, instance:com.actitime.testscripts.CustomerTest@4501b7af]" name="loginApp" is-config="true" duration-ms="12046" started-at="2020-06-19T06:35:16Z" finished-at="2020-06-19T06:35:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginApp -->
        <test-method status="FAIL" signature="testCreateCustomer()[pri:0, instance:com.actitime.testscripts.CustomerTest@4501b7af]" name="testCreateCustomer" duration-ms="11571" started-at="2020-06-19T06:35:28Z" finished-at="2020-06-19T06:35:40Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Bala Sai Krishna] but found [All Customers]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Bala Sai Krishna] but found [All Customers]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.actitime.testscripts.CustomerTest.testCreateCustomer(CustomerTest.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateCustomer -->
        <test-method status="PASS" signature="logoutApp()[pri:0, instance:com.actitime.testscripts.CustomerTest@4501b7af]" name="logoutApp" is-config="true" duration-ms="5486" started-at="2020-06-19T06:35:40Z" finished-at="2020-06-19T06:35:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutApp -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.actitime.testscripts.CustomerTest@4501b7af]" name="closeBrowser" is-config="true" duration-ms="97" started-at="2020-06-19T06:35:45Z" finished-at="2020-06-19T06:35:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.actitime.testscripts.CustomerTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
